web/pdf_viewer.js

```diff
@@ -2204,17 +2204,22 @@ class PDFViewer {
     ]);
   }
 
-  /**
-   * @type {number}
-   */
   get annotationEditorMode() {
     return this.#annotationEditorUIManager
       ? this.#annotationEditorMode
       : AnnotationEditorType.DISABLE;
   }
 
   /**
-   * @param {number} mode - AnnotationEditor mode (None, FreeText, Ink, ...)
+   * @typedef {Object} AnnotationEditorModeOptions
+   * @property {number} mode - The editor mode (none, FreeText, ink, ...)
+   * @property {string|null} [editId] - ID of the existing annotation to edit
```

Comment:
Nit: A somewhat pre-existing issue, but please add periods at the end of these sentences.